catalog/async/catalog-async-data/pom.xml

```diff
@@ -73,17 +73,17 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.70</minimum>
+                                            <minimum>0.75</minimum>
```

Comment:
The message about the low ratios can probably be removed.

Reply:
Removed message